tests/zfs-tests/tests/functional/cli_root/zdb/zdb_recover.ksh

```diff
@@ -0,0 +1,55 @@
+#!/bin/ksh
```

Comment:
Can you change the permissions on the script to `755`.  This is what caused the style failure.